### DISCUSSION TITLE
Replacement for Sidekiq.redis_version in sidekiq-pro 7.0

### DISCUSSION THREAD

**Discussion by leifg** [sidekiq/sidekiq#5612] [Q&A] [answered]
We've been using `Sidekiq.redis_version` in one of our health controller to verify the sidekiq redis connection is working (We .

Looks like this method has been removed in sidekiq-pro 7.0 so I am wondering if there is an alternative or a better way to verify the redis connection?

**Comment by mperham** [comment]
You can take the previous implementation and stash it somewhere as a utility method in your app code. I didn't want to provide a global API for it anymore. Or with redis_client, it's as simple as:

```ruby
require "redis_client"
RedisClient.new.call("INFO").split.select {|line| line =~ /^redis_version/ }
```

**Comment by leifg** [comment, marked as answer]
I ended up using `Sidekiq.redis(&:info)` as used by [health-monitor-rails](https://github.com/lbeder/health-monitor-rails/blob/master/lib/health_monitor/providers/sidekiq.rb#L100)
